dev/stacks/dpe-sandbox-k8s/main.tf

```diff
@@ -1,16 +1,23 @@
 module "sage-aws-vpc" {
-  source   = "spacelift.io/sagebionetworks/sage-aws-vpc/aws"
-  version  = "0.2.2"
-  vpc_name = "dpe-sandbox"
+  source             = "spacelift.io/sagebionetworks/sage-aws-vpc/aws"
+  version            = "0.3.3"
+  vpc_name           = "dpe-sandbox"
+  capture_flow_logs  = true
+  flow_log_retention = 1

```

Comment:
do we want to increase retention of logs to e.g. 3 days in case we have things running and breaking over the weekends?

Reply:
For dev I don't think we will need to. However, yes - We will want to use the retention period that is higher than a day. I'm not sure at this point what that period should be, but it's something I will determine before we move over to prod.

Reply:
Word on the street this might be 90 days. Still looking to confirm with IT on this